src/soss.md

```diff
@@ -1,5 +1,11 @@
 # SOSS
 
-The system-of-systems synthesizer (SOSS) is a tool which provides protocol
-translation between different subsystems. Currently, ROS 2, ROS 1, and
-Websockets are supported. Such composite systems can be called The ROS-SOSS.
+The ecosystem of different message passing systems is vast and diverse. Without any one single system that is unanimously considered the best for all applications, we are left to consider how we can tie together disparate message passing systems to bridge the gap between the different kinds of applications that all play a critical role in a modern, intelligent robotics solution. The best protocols to use for inter-robot communication might not be the best for remote operator communication or for end-user communication.
```

Comment:
```suggestion
The system-of-systems synthesizer (SOSS) is a tool which provides protocol translation between different subsystems. Such composite systems can be called The ROS-SOSS. To see the current implementation status, see the [SOSS repository](https://github.com/osrf/soss). 

##Motivation and Introduction

The ecosystem of different message passing systems is vast and diverse. Without any one single system that is unanimously considered the best for all applications, we are left to consider how we can tie together disparate message passing systems to bridge the gap between the different kinds of applications that all play a critical role in a modern, intelligent robotics solution. The best protocols to use for inter-robot communication might not be the best for remote operator communication or for end-user communication.
```

Reply:
The other sections typically have an overview header and then some details so I've re-worded the overview and inserted a section title here. Also we don't have any link to the SOSS repo so this seemed like a good place to put it.